src/html_json.js

```diff
@@ -181,12 +157,34 @@ module.exports.main = async (context, action) => {
   const {
     owner, repo, ref, path,
   } = action.request.params;
+
   const { logger } = action;
 
+  const secrets = setdefault(action, 'secrets', {});
+  secrets.REPO_RAW_ROOT = secrets.REPO_RAW_ROOT || 'https://raw.githubusercontent.com/';
+  secrets.HTTP_TIMEOUT = secrets.HTTP_TIMEOUT || 10000;
+
+  const loadConfig = async () => {
+    const downloader = new Downloader(context, action, {
```

Comment:
why not use `action.downloader` ?

Reply:
Didn't know, but what about `forceHttp1`, can I somehow pass that option?

Reply:
for the tests, use `process.env.HELIX_PIPELINE_FORCE_HTTP1`

see: https://github.com/adobe/helix-pipeline/blob/ec0995b82728d5956dbdb551d1089c7938c9b561/src/utils/Downloader.js#L36